bedrock/settings/base.py

```diff
@@ -535,9 +535,9 @@ def get_app_name(hostname):
     return "bedrock"
 
 
-HOSTNAME = platform.node()
-APP_NAME = get_app_name(HOSTNAME)
+APP_NAME = config("APP_NAME", default="bedrock-noenvironment")
```

Comment:
I would leave `HOSTNAME` above this and have it be the default here, for a smooth transition where it still works for the old clusters.

Reply:
Updated 👍